code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -67,13 +67,13 @@
 		PF.flags_can_pass_all = PASS_ALL^PASS_OVER_THROW_ITEM
 
 /mob/living/carbon/xenomorph/facehugger/Life(delta_time)
+	if(stat != DEAD && !lying && !(mutation_type == FACEHUGGER_WATCHER) && !(locate(/obj/effect/alien/weeds) in get_turf(src)))
+		adjustBruteLoss(1)
+		return
```

Comment:
This has stopped facehuggers losing health at all when off weeds from what I can see.

Reply:
Going to check if this is the case but the "!(mutation_type == FACEHUGGER_WATCHER)" part of the if statement should only leave the watcher alone as intended because it should only pass these through if they are NOT the watcher.

Reply:
Yeah that's what it is. So it isn't that you don't take damage it's that the status bug doesn't update your damage until your status changes vs regularly 

See this GIF as it shows how it updates the damage all at once a few seconds after you get back on weeds.

![dreamseeker_J1MEMbb6vs](https://github.com/cmss13-devs/cmss13/assets/24533979/9ceebb49-110f-4dc9-a7af-3a9e86d8d699)

Reply:
I got reports in dchat so it's not just status bar bug. You can run without weeds for infinite amount of time, then rest and explode. So you did broke something.

Reply:
Probably because you added `return` instead of `return ..()`

Reply:
https://github.com/cmss13-devs/cmss13/issues/4062

Reply:
You basically don't call parent because of this return so everything is broken.

Reply:
Fixed; good catch @ihatethisengine